src/main/java/cart/domain/member/domain/Member.java

```diff
@@ -0,0 +1,70 @@
+package cart.domain.member.domain;
+
+public class Member {
+    private static final int INITITAL_CASH = 5000;
```

Comment:
```suggestion
    private static final int INITITAL_CASH = 5_000;
```
위와 같이 작성하면 보기가 더 편하더라구요~

Reply:
오! 럿고 덕분에 이렇게 해도 동작하는지 처음 알았습니다!! 감사합니다 ㅎㅎㅎ